Yosemite/Yosemite/Stores/MediaStore.swift

```diff
@@ -159,6 +165,24 @@ private extension MediaStore {
         }
     }
 
+    func uploadFile(siteID: Int64,
```

Comment:
nit: Consider adding a few unit tests for this new action. Considering that we are firing `uploadMedia` which already has enough unit tests maybe testing basic success and error cases should be enough.